Registration/RegisterWithAzure.psm1

```diff
@@ -1622,7 +1642,10 @@ function Remove-RegistrationResource{
 [CmdletBinding()]
     param(
         [Parameter(Mandatory = $true)]
-        [String] $ResourceId
+        [String] $ResourceId,
```

Comment:
can result in failures as its a mandatory parameter

Reply:
I didn't change this. But I think ResourceId being mandatory is correct , require to remove the resource identified.